src/cli/task.rs

```diff
@@ -182,6 +192,11 @@ pub struct Args {
     pub manifest_path: Option<PathBuf>,
 }
 
+fn print_heading(value: &str) {
+    let bold = console::Style::new().bold();
+    println!("{}\n{:-<2$}", bold.apply_to(value), "", value.len() + 4);
```

Comment:
https://github.com/prefix-dev/pixi/blob/f7dd0086be4a27f5530755154f483514aafa24cd/src/cli/list.rs#L239

Maybe it would be nicer to use a tabwriter like we do in `pixi list` and some more places. So you don't hardcode the initial indent. 

silly example of where the hardcoded width might be not so nice:
```
➜ pixi task list -s
Tasks per environment:
--------------------------
           default: build, build-docs, bump, deploy-dev, deploy-latest, docs, generate-schema, install, lint, mike-serve, pypi-proxy, test, test-all, test-schema, toml-format, toml-lint, 
              docs: build-docs, deploy-dev, deploy-latest, docs, mike-serve, 
blablablablablablablabla: generate-schema, test-schema, toml-format, toml-lint, 
```

Reply:
have changed the behaviour as suggested, output now

```
Tasks per environment:
--------------------------
default                     : build, build-docs, bump, deploy-dev, deploy-latest, docs, generate-schema, install, lint, mike-serve, pypi-proxy, test, test-all, test-schema, toml-format, toml-lint, 
docs                        : build-docs, deploy-dev, deploy-latest, docs, mike-serve, 
schemaaaaaaaaaaaaaaaaaaaaa  : generate-schema, test-schema, toml-format, toml-lint, 
```